tests/integration/pdo/mysql/base-class/test_instance_reporting_1.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+When PDO base class constructor is used to create connection object
+to a database on a remote host via a TCP port,
+and database instance reporting is enabled, the agent should
+ - not generate errors
+ - record a datastore metrics
```

Comment:
Copy/paste comment occurs 27 times.
Suggestion to change:
`record a datastore metrics`
to be:
`record datastore metrics`

Reply:
Fixed in 9bb85e4fe01202286e4323acb21a212601e82b6e